packages/expo-image/android/src/main/java/expo/modules/image/ExpoImageModule.kt

```diff
@@ -29,16 +32,24 @@ class ExpoImageModule : Module() {
   override fun definition() = ModuleDefinition {
     Name("ExpoImage")
 
-    AsyncFunction("prefetch") { urls: List<String>, cachePolicy: CachePolicy, promise: Promise ->
+    AsyncFunction("prefetch") { urls: List<String>, cachePolicy: CachePolicy, headersMap: ReadableMap?, promise: Promise ->
```

Comment:
```suggestion
    AsyncFunction("prefetch") { urls: List<String>, cachePolicy: CachePolicy, headersMap: Map<String, String>?, promise: Promise ->
```
You can use `Map` here instead, which is much better because we're checking if the values provided by JS are correct. The API is doing the validation, so you don't have to.

Reply:
Thanks for your review! I changed the code.